app/helpers/spree/admin/digital_helper.rb

```diff
@@ -0,0 +1,28 @@
+module Spree
+  module Admin
+    module DigitalHelper
+      def asset_icon(asset)
+        icon_size = 50
+
+        case asset.attachment.content_type
+        when 'application/pdf'
+          svg_icon name: 'file-earmark-pdf.svg', width: icon_size, height: icon_size
+        when 'image/png', 'image/jpeg', 'image/jpg', 'image/gif', 'image/bmp', 'image/tiff', 'image/svg+xml'
```

Comment:
@MatthewKennedy I think a regexp would work better here to capture all image formats